modules/power-monitoring-configuring-kepler-redfish.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/configuring-power-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="power-monitoring-configuring-kepler-redfish_{context}"]
+= Configuring {PM-kepler} to use Redfish
+
+You can configure the {PM-kepler} to use the Redfish as the source for platform power.
+
+.Prerequisites
+* You have access to the {product-title} web console.
+* You are logged in as a user with the `cluster-admin` role.
+* You have installed the {PM-operator}.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, go to *Operators* -> *Installed Operators*.
+
+. Click *{PM-title-c}* from the *Installed Operators* list and go to the *{PM-kepler}* tab.
+
+. Click *Create {PM-kepler}*. If you already have a {PM-kepler} instance created, click *Edit Kepler*.
+
+. Configure {PM-kepler} instance’s `.spec.exporter.redfish` specifying the mandatory `secretRef` and optional `probeInterval` and `skipSSLVerify`.
++
+.Example output
+[source,yaml]
+----
+apiVersion: kepler.system.sustainable.computing.io/v1alpha1
+kind: Kepler
+metadata:
+  name: kepler
+spec:
+  exporter:
+    deployment:  ... 
```

Comment:
I'm not sure what the three dots mean here. The "Example Kepler resource with default configuration" uses a different approach. How about:

deployment:
# ...

Reply:
So, under "deployment", you got 3 more fields which isn't expanded here. The engineering team only wants to show the fields under "redfish". FYI, both are optional tasks and not mandatory for users to perform and Kepler would still be created without specifying "deployment" or "redfish".